crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -795,6 +802,38 @@ pub fn tile_config(node: &Node) -> TileConfig {
     TileConfig::new(repeat)
 }
 
+/// Create a TopKConfig from the attributes of the node.
+pub fn top_k_config(node: &Node) -> TopKConfig {
+    // extract the shape of the input data tensor
+    let data_tensor = match node.inputs.first().unwrap().clone().ty {
+        ArgType::Tensor(tensor) => tensor,
+        _ => panic!("Only tensor input is valid"),
+    };
+
+    let k = match node.inputs.get(1) {
+        Some(k_tensor) => k_tensor
+            .clone()
+            .value
+            .expect("Expecting K tensor to have a value.")
+            .into_i64s()[0],
+        _ => extract_attr_value_i64(node, "k"),
+    };
+
+    let mut axis: i64 = extract_attr_value_i64(node, "axis");
+
+    // if axis is negative, it is counted from the end
+    if axis < 0 {
+        axis += data_tensor.dim as i64;
+    }
+
+    let largest = match node.attrs.get("largest") {
+        Some(val) => val.clone().into_i64(),
+        _ => 1,
+    };
```

Comment:
If you're not checking for "k" as the second input of the node (for opsets 10, 11) and just adding support for opset 1, then we don't need to check for the "largest" attribute here. It's only present in the later version 11 of the op.

So we can remove this from the config and node.

Reply:
sure thing will remove and resubmit tonight